packages/core/src/analytics/AnalyticsManager.ts

```diff
@@ -37,11 +44,19 @@ export class AnalyticsManager {
     apiKey,
     app,
     cartridgeTag,
-  }: { apiUri?: string; apiKey: string; app: AppInfo; cartridgeTag: string }) {
+    device,
+  }: {
+    apiUri?: string;
+    apiKey: string;
+    app: AppInfo;
+    cartridgeTag: string;
+    device?: Device;
```

Comment:
Is this something we can detect ourselves instead of requiring the consumer to pass in?